src/elona/data/types/type_god.hpp

```diff
@@ -8,14 +8,20 @@
 namespace elona
 {
 
+struct GodSkillBonus
+{
+    int factor;
+    int max;
+};
+
 struct GodData
 {
     int id;
     optional<std::string> wish_name;
     optional<SharedId> summon_id;
     sol::table items;
     SharedId servant_id;
-    std::unordered_map<int, lua::WrappedFunction> skills;
+    std::unordered_map<int, GodSkillBonus> skills;
```

Comment:
In this commit, WrappedFunction was replaced by GodSkillBonus, but include directive for wrapped_function.hpp still remain.